corehq/apps/accounting/forms.py

```diff
@@ -121,7 +119,7 @@ def __init__(self, account, *args, **kwargs):
                 'name': account.name,
                 'salesforce_account_id': account.salesforce_account_id,
                 'currency': account.currency.code,
-                'emails': contact_info.emails,
+                'email_list': ','.join(contact_info.email_list),
```

Comment:
`email_list` could be a bit confusing considering it's a string not a list, but admittedly don't really have a better name for it